DragaliaAPI.Shared.Test/Unit/MasterAssetTest.cs

```diff
@@ -477,4 +477,231 @@ public void StoryData_Type_IsCorrect(int storyId, StoryTypes expectedType)
     {
         MasterAsset.MasterAsset.UnitStory[storyId].Type.Should().Be(expectedType);
     }
+
+    [Fact]
+    public void AbilityCrestTrade_Get_ReturnsExpectedProperties()
+    {
+        AbilityCrestTrade abilityCrestTrade = MasterAsset.MasterAsset.AbilityCrestTrade.Get(5101);
+
+        abilityCrestTrade
+            .Should()
+            .BeEquivalentTo(
+                new AbilityCrestTrade(
+                    Id: 5101,
+                    AbilityCrestId: AbilityCrests.SweetSurprise,
+                    NeedDewPoint: 4000,
+                    Priority: 5199
+                )
+            );
+    }
+
+    [Fact]
+    public void AbilityCrestTrade_Enumerable_ReturnsListOfAbilityCrestTrades()
+    {
+        IEnumerable<AbilityCrestTrade> abilityCrestTrades = MasterAsset
+            .MasterAsset
+            .AbilityCrestTrade
+            .Enumerable;
+
+        abilityCrestTrades.Should().AllBeOfType<AbilityCrestTrade>();
+    }
```

Comment:
Not sure what the value of this test is

Reply:
uh. i'm ngl i forgor lmao i'll just remove it

Reply:
Yes please